### PR TITLE
Minikube specific example

### PR DESCRIPTION
**Pull request by rbankston**
This pull request uses nip.io so users aren't forced to change their /etc/hosts file to see that ingress to a specific host works properly. Added an example yaml that I verified on my local minikube cluster and an updated README after the dns portion about that example.

### CODE REVIEW

**Comment by jbeda** [comment]
Interesting! 

@Bradamant3 -- can you take a look at this to see how it fits in with the README?  Not sure if we want to expand the readme or have a dedicated minikube page.

Also -- you need to put in the "Signed-off-by" line as specified in CONTRIBUTING.md.

**Comment by Bradamant3** [comment]
A separate topic/page/markdown for Minikube would be better UX. @rbankston if you want to fix the "Signed-off-by" issue I can merge (if I have permissions ...) and rework the content.

**Comment by davecheney** [comment]
@rbankston thanks for your PR. To make the DCO bot happy the simplest thing might be to rebase against master and squash your commit down to one, then amend it with signed off. Something like this (typed from memory) should do the job.
```
git checkout -b $YOURBRANCH
git fetch origin master
git rebase -i origin/master
// squash one commit into the previous one
git commit --amend -s
// fix up the commit message, make sure the Signed-Off: line is intact.
git push -f
```

File: README.md [hunk 1]
```diff
@@ -80,7 +80,15 @@ How you configure DNS depends on your platform:
 
 - On AWS, create a CNAME record that maps the host in your Ingress object to the ELB address.
 - If you have an IP address instead (on GCE, for example), create an A record.
-- On Minikube, you can fake DNS by editing `/etc/hosts`.
+- On Minikube, you can fake DNS by editing `/etc/hosts` or you can use the provcided example and not have to modify dns on your local machine.
```
Comment (davecheney) [inline README.md]: s/the provcided example/this example to avoid modifying your dns settings.

**Comment by davecheney** [comment]
Hi, 

I tried to rebase this and fix up the final edits in an effort to merge it but I found the rebandly link was broken.

I'm sorry this PR has taken so long to merge, and due to the DCO requirement I can't just edit your commit and merge it.

Can you please send a PR with the minikube deployment so we don't have a random rebrandly link pointing somewhere that isn't controlled by this project.

Thanks

Dave

**Comment by jbeda** [comment]
If anyone needs a j.hept.io link just slack me.

**Comment by rbankston** [comment]
@davecheney the link exists once merged

**Comment by davecheney** [comment]
Thank you. sorry this too so long to merge.

**Pull request merged**